pages/composition-api.vue

```diff
@@ -81,7 +82,13 @@ export default defineComponent({
         .then((data: ToDo[]) => { fetchedTodos.value = data })
     })
 
-    useMeta({ title: 'Composition API Demo' })
+    useMeta({
+      title: 'Composition API Demo',
+      meta: [{
+        name: 'message',
+        content: message.value // It's not reactive
+      }]
+    })
```

Comment:
@hmsk Sorry for the delay in replying. I'm afraid this isn't actually that intuitive at the moment. I might need to add the functionality to `@nuxtjs/composition-api`.

Reply:
Does this API look good to you?

https://github.com/nuxt-community/composition-api/pull/255

Reply:
Makes sense for me. `useMeta(() => ({}))` is intuitive and looks similar to the type of `head ()` in options/class API.

Reply:
a812357 😉